README.md

```diff
@@ -30,14 +35,135 @@ And then execute:
+automatically, but in some cases (specially when actively changes that type of files) you want to 
+speed up the process. 
+
+To do that, start in a separate terminal than the one with `bin/rails s`, and BEFORE it, the following command:
+
+```
+bin/webpack-dev-server
+```
+
+#### Code Styling
+
+Please follow the code styling defined by the different linters that ensure we
+are all talking with the same language collaborating on the same project. This
+project is set to follow the same rules that Decidim itself follows.
+
+[Rubocop](https://rubocop.readthedocs.io/) linter is used for the Ruby language.
+
+You can run the code styling checks by running the following commands from the
+console:
+
+```
+$ bundle exec rubocop
+```
+
+To ease up following the style guide, you should install the plugin to your
+favorite editor, such as:
+
+- Atom - [linter-rubocop](https://atom.io/packages/linter-rubocop)
+- Sublime Text - [Sublime RuboCop](https://github.com/pderichs/sublime_rubocop)
+- Visual Studio Code - [Rubocop for Visual Studio Code](https://github.com/misogi/vscode-ruby-rubocop)
+
+#### Non-Ruby Code Styling
+
+There are other linters for Javascript and CSS. These run using NPM packages. You can
+run the following commands:
+
+1. `npm run lint`: Runs the linter for Javascript files.
+2. `npm run lint-fix`: Automatically fix issues for Javascript files (if possible).
+3. `npm run stylelint`: Runs the linter for SCSS files.
+4. `npm run stylelint-fix`: Automatically fix issues for SCSS files (if possible).
+
+### Testing
+
+To run the tests run the following in the gem development path:
+
+```bash
+$ bundle
+$ DATABASE_USERNAME=<username> DATABASE_PASSWORD=<password> bundle exec rake test_app
+$ DATABASE_USERNAME=<username> DATABASE_PASSWORD=<password> bundle exec rspec
+```
+
+Note that the database user has to have rights to create and drop a database in
+order to create the dummy test app database.
+
+In case you are using [rbenv](https://github.com/rbenv/rbenv) and have the
+[rbenv-vars](https://github.com/rbenv/rbenv-vars) plugin installed for it, you
+can add these environment variables to the root directory of the project in a
+file named `.rbenv-vars`. In this case, you can omit defining these in the
+commands shown above.
+
+### Test code coverage
+
+Code coverage report is generated automatically in a folder named `coverage` in the project root which contains
+the code coverage report.
+
+```bash
+firefox coverage/index.html
+```
+
+### Localization
+
+If you would like to see this module in your own language, you can help with its
+translation at Crowdin:
+
+https://crowdin.com/project/decidim-module-vocdoni
+
 ## License
 
 This engine is distributed under the GNU AFFERO GENERAL PUBLIC LICENSE.
+See [LICENSE-AGPLv3.txt](LICENSE-AGPLv3.txt).
+
+As this module works with the Vocdoni App, see https://vocdoni.app/ for more information about their open source licenses.
```

Comment:
Just a couple corrections regarding this line:

1. We're actually using the Vocdoni API/SDK, so we should link to https://vocdoni.io/  (I corrected that in other PR) 
2. Regarding their license, it's not relevant for us, as using an API it's not considered a derivative work. See https://opensource.stackexchange.com/a/8204 (They're using the AGPL for their SDK https://github.com/vocdoni/vocdoni-sdk/blob/main/LICENSE)

Reply:
Ok, I guess changing the link is enough?

Reply:
Mmm yes, let's change the link and the name too